spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -145,7 +145,7 @@
       it "log an error message and throw RuntimeError if it can't find the indicated whitelist file" do
         VCR.use_cassette('cant_find_whitelist_call') do
           exp_msg = 'Unable to find list of druids at bad_path'
-          indexer = Harvestdor::Indexer.new(@config.merge(:whitelist => 'bad_path'))
+          indexer = described_class.new(@config.merge(whitelist: 'bad_path'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.